src/index.html

```diff
@@ -10,8 +10,213 @@
+            id="email"
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label for="password" class="label">
+          Password:
+          <input id="password"
+            type="password"
+            name="password"
+            minlength="8"
+            maxlength="16"
+            autocomplete="off"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="label">
+          Do you love cats?
+          <label for="cats">
+            <label for="radiobutton">
+              <input
+                type="radio"
+                name="cats"
+                value="Yes"
+                required
+                autocomplete="off"
+              >
+              Yes
+            </label>
+          </label>
+          <label for="cats">
+            <label for="radiobutton">
+              <input
+                type="radio"
+                name="cats"
+                value="No"
+                required
+                autocomplete="off"
+              >
+              No
+            </label>
+          </label>
+        </div>
+
+        <label for="favoriteColor" class="label">
+          What is your favorite color?
+          <input
+            id="favoriteColor"
+            type="color"
+            name="color"
+            required
+            autocomplete="off"
+          >
+        </label>
+
+        <label for="goToBed"
+          class="label">
+          What time do you go to bed?
+          <input
+            id="goToBed"
+            type="time"
+            name="goToBed"
+            placeholder="--:--:-- --"
+            required
+            autocomplete="off"
+          >
+        </label>
+
+        <label for="favoriteBrandOfCar"
+          class="label">
+          What are your favorite brand of cars?
+          <select name="favoriteBrandOfCar" id="favoriteBrandOfCar"
+            multiple
+            required
+            autocomplete="off">
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+            <option value="placeholder"></option>
+          </select>
+        </label>
+
+        <label for="rate"
+        class="label">How do you rate our work?
+          <input
+            id="rate"
+            type="range"
+            min="1"
+            max="10"
+            required
+            autocomplete="off"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info</legend>
+        <label for="comments"
+        class="label">Comments:
+          <textarea name="comments"
+            id="comments"
+            cols="20" rows="2"
+          >
+          </textarea>
```

Comment:
```suggestion
          ></textarea>
```
to fix this
<img width="297" alt="image" src="https://user-images.githubusercontent.com/53611910/200178830-8db138f7-dbc6-4ec8-8f25-c63b78151122.png">